### ISSUE TITLE
setting isEditable to true is not working

### ISSUE THREAD

**Issue by cedetel** [scalar/scalar#4078]
### What happens?

When setting the `isEditable` to true in configuration, the swagger editor is not shown. Only blank screen is shown in the middle. 

Here is my code:
```
<!doctype html>
<html>
  <head>
    <title>Scalar API Reference</title>
    <meta charset="utf-8" />
    <meta
      name="viewport"
      content="width=device-width, initial-scale=1" />
  </head>
  <body>
    <script
      id="api-reference"
      data-url="https://cdn.jsdelivr.net/npm/@scalar/galaxy/dist/latest.yaml">
      </script>

    <!-- Optional: You can set a full configuration object like this: -->
    <script>
      var configuration = {
        theme: 'purple',
        isEditable: true
      }

      document.getElementById('api-reference').dataset.configuration =
        JSON.stringify(configuration)
    </script>

    <script src="https://cdn.jsdelivr.net/npm/@scalar/api-reference"></script>
  </body>
</html>
```

### What did you expect to happen?

setting the isEditable to true in configuration should show the swagger editor

.

### OpenAPI Document

https://cdn.jsdelivr.net/npm/@scalar/galaxy/dist/latest.yaml

**Comment by marcosmartini** [comment]
Same here. We're using a pinned CDN version and the `isEditable` is broken too. 

Does this mean that even if we're using a pinned CDN version, other packages used by the pinned version are not actually pinned, therefore potentially introducing bugs like this one?

Thanks for the great product, by the way

**Comment by hi-otto** [comment]
+1, Thanks for the excellent product, I hope someone can fix it

**Comment by KylinWu** [comment]
I'm experiencing the same issue . Any guidance on proper configuration or troubleshooting would be appreciated!

**Comment by iamsid** [comment]
@amritk Hi. Any update on this? Shows the same blank UI in the latest version 1.28.32 as well.

**Comment by amritk** [comment]
So this was actually changed a while ago, I am updating the documentation right now. Since shipping the editor with every build was unnecessarily costly, now `isEditable` just adds some css. You still need to inject the editor yourself.

We may add this as a plugin down the line but for you I can give you two examples of places we do it:
- heres the code for our [sandbox](https://sandbox.scalar.com/) https://github.com/scalar/sandbox/blob/main/src/App.vue
- and another editor we have in our [examples](https://github.com/scalar/scalar/blob/main/examples/web/src/pages/ApiReferencePage.vue)

Sorry its no longer as easy as flicking a switch!

**Comment by Ulis123** [comment]
Hi @amritk . I read the documentation for the plugins `https://github.com/scalar/scalar/blob/main/documentation/plugins.md`, and I didn't get how the plugins system works. There is no example for a working plugin. Is it working the same as in SwaggerUI?

I am using React 19. Can you explain or give me a code example to add an Editor to the Scalar?

**Comment by amritk** [comment]
hey @Ulis123 I'm not sure if its fully fleshed out, @hanspagel would know. But the example we have can be found here:

https://github.com/scalar/scalar/tree/main/packages/api-reference/playground/vue/src/x-custom-extension-plugin

**Comment by Ulis123** [comment]
@amritk Yes I tested the plugin system and it seems it's adding markup only to one block under the info block. I didn't find a way to change the location where I can add my plugin. So I still don't understand how to add my Editor with React. Also, I don't have access to the scalar's state in the plugins, that's also not so usefull.
